Servers/services/email/types.ts

```
@@ -0,0 +1,233 @@
+    return false;
+  }
+
+  // Prevent dangerous HTML/Script characters that could cause injection
+  const dangerousChars = /[<>'"()\\[\]{}]/;
+  if (dangerousChars.test(email)) {
+    return false;
+  }
+
+  // Prevent Unicode characters that could cause homograph attacks
+  // Only allow ASCII characters
+  if (!/^[\x00-\x7F]*$/.test(email)) {
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)